portalnet/src/overlay_service.rs

```diff
@@ -2753,9 +2753,11 @@ mod tests {
         storage::{DistanceFunction, MemoryContentStore},
         utils::db::setup_temp_dir,
     };
-    use ethportal_api::types::content_key::overlay::IdentityContentKey;
     use ethportal_api::types::distance::XorMetric;
     use ethportal_api::types::enr::generate_random_remote_enr;
+    use ethportal_api::types::{
```

Comment:
*nitpick* Our general import statement ordering standard is poorly defined, mostly due to there not being any solid linting options available on the `stable` rust channel. But, in this case I would prefer either...
```rust
use ethportal_api::types::{
    cli::DEFAULT_DISCOVERY_PORT,
    content_key::overlay::IdentityContentKey,
    distance::XorMetric,
    enr::generate_random_remote_enr,
};
```
or...
```rust
use ethportal_api::types::cli::DEFAULT_DISCOVERY_PORT;
use ethportal_api::types::content_key::overlay::IdentityContentKey;
use ethportal_api::types::distance::XorMetric;
use ethportal_api::types::enr::generate_random_remote_enr;
```

Reply:
Done. I tend not to look at imports and consider anything that automatic formatting / tooling does as good enough (especially if clippy doesn't complain).